gf-ui-lib/src/components/Map/mapLeaflet/markersLeaflet/markerLeaflet/MarkerLeaflet.tsx

```diff
@@ -0,0 +1,39 @@
+import {DivIcon, Icon, IconOptions, LatLngExpression, PointExpression} from "leaflet";
+import {useEffect, useRef} from "react";
+import {Marker, Popup} from "react-leaflet";
+
+interface MarkerLeafletProps {
+  markerPosition: LatLngExpression
+  markerIcon: Icon<IconOptions> | DivIcon | undefined
+  eventHandlers: object
+  popupOffset: PointExpression | undefined
+  popupText: string
+  openPopup: boolean
```

Comment:
Can you rename this property to popUpOpen? Because all other properties start with popUp

Reply:
Done